src/Sylius/Bundle/ShopBundle/Resources/views/Taxon/_breadcrumb.html.twig

```diff
@@ -1,15 +1,15 @@
+{% from '@SyliusShop/Common/Macro/breadcrumb.html.twig' import breadcrumb %}
 {% set ancestors = taxon.ancestors|reverse %}
-
-<div class="ui breadcrumb">
-    <a href="{{ path('sylius_shop_homepage') }}" class="section">{{ 'sylius.ui.home'|trans }}</a>
-    <div class="divider"> / </div>
-    {% for ancestor in ancestors %}
-        {% if ancestor.isRoot() %}
-            <div class="section">{{ ancestor.name }}</div>
-        {% else %}
-            <a href="{{ path('sylius_shop_product_index', {'slug': ancestor.slug, '_locale': ancestor.translation.locale}) }}" class="section">{{ ancestor.name }}</a>
-        {% endif %}
-    <div class="divider"> / </div>
-    {% endfor %}
-    <div class="active section">{{ taxon.name }}</div>
-</div>
+{% set crumbs = [] %}
+{% for ancestor in ancestors %}
+    {% if not ancestor.isRoot() %}
+        {% set crumbs = crumbs|merge([{
+            url: path('sylius_shop_product_index', {'slug': ancestor.slug, '_locale': ancestor.translation.locale}),
+            name: ancestor.name
+        }]) %}
+    {% endif %}
+{% endfor %}
```

Comment:
If above logic will be extracted to Twig function, here it could be reused probably

Reply:
WDYT if it comes to something like this (take the idea, not the implementation itself):

```suggestion
{% set crumbs = getCrumbsFromTaxons(ancestors) %}
```

Would it be ok to you?